catchup/catchpointService.go

```diff
@@ -472,18 +472,10 @@ func lookbackForStateproofsSupport(topBlock *bookkeeping.Block) uint64 {
 	if proto.StateProofInterval == 0 {
```

Comment:
In the comment above: lookbackForStateproofSupport -> lookbackForStateproofsSupport